api/api_keys/user.py

```diff
@@ -18,11 +19,16 @@
     from projects.models import Project
 
 
-class APIKeyUser:
+class APIKeyUser(UserABC):
     def __init__(self, key: MasterAPIKey):
         self.key = key
 
-    def is_authenticated(self):
+    @property
+    def is_authenticated(self) -> bool:
+        return True
+
+    @property
+    def is_master_api_key_user(self) -> bool:
         return True
```

Comment:
Does this need to be a property? Can we not set it directly in the `__init__` method?

Reply:
It just mimics the style of django I think, no? 
e.g:

```
    @property
    def is_anonymous(self):
        return True

    @property
    def is_authenticated(self):
        return False


```

Reply:
Ok, sure. I guess they did that though so that you could add run time logic to decide the outcome - I'm not sure we'll ever need it here. But I don't think it's an issue anyway.